packages/dropdowns.next/src/index.ts

```diff
@@ -24,5 +28,9 @@ export type {
   IOptGroupProps,
   ITagProps,
   ISelectedOption,
-  OptionValue
+  OptionValue,
+  IItemProps,
+  IItemGroupProps,
+  IMenuProps,
+  OptionValue as ItemValue
```

Comment:
I'll comment these out prior to merge so we don't leak unsupported component exports into production.